js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -177,3 +202,438 @@ category=Kategorie
+executeActions=Ausführen von Aktionen
+clientPolicies=Client Richtlinien
+profiles=Profile
+policies=Richtlinien
+newClientProfile=Client Profil erstellen
+profilesConfigType=Konfigurieren mit\:
+policiesConfigTypes.formView=Formular
+policiesConfigTypes.jsonEditor=JSON Editor
+noClientPolicies=Keine Client Richtlinien vorhanden
+noClientPoliciesInstructions=Es sind keine Kundenrichtlinien vorhanden. Wählen Sie "Client-Richtlinie erstellen", um eine neue Client-Richtlinie zu erstellen.
+createPolicy=Client-Richtlinie erstellen
+clientPolicySearch=Client-Richtlinie suchen
+createClientPolicyError=Richtlinie konnte nicht erstellt werden aufgrund von\: {{error}}
+userProfile=Benutzerprofil
+attribute=Attribut
+attributesGroup=Attributgruppe
+allGroups=Alle Gruppen
+createAttribute=Attribut erstellen
+createGroupText=Attributgruppe erstellen
+attributeDisplayName=Anzeigename
+columnDisplayDescription=Beschreibung
+annotations=Annotationen
+hideInheritedRoles=Geerbte Rollen ausblenden
+assignRole=Rolle zuweisen
+searchByName=Suche nach Namen
+searchByRoleName=Suche nach Rollennamen
+unAssignRole=Rolle entfernen
+whatIsDefaultGroups=Was ist die Funktion von Standardgruppen?
+noDefaultGroupsInstructions=Mit Standardgruppen können Sie automatisch eine Gruppenzugehörigkeit zuweisen, wenn ein neuer Benutzer erstellt oder während des <1>Identitätsbrokerings</1> importiert wird.
+noDefaultGroups=Keine Standardgruppen definiert
+authenticationExplain=Im Bereich Authentifizierung können Sie verschiedene Arten von Anmeldeinformationen konfigurieren und verwalten.
+flows=Abläufe
+searchForFlow=Suche nach Ablauf
+createFlow=Ablauf erstellen
+authenticationCreateFlowHelp=Ablauf erstellen
+flowName=Ablaufname
+flowType=Ablauftyp
+usedBy=Vewendet von
+duplicate=Duplizieren
+setAsDefaultAction=Als Standardaktion festlegen
+authDefaultActionTooltip=Wenn diese Option aktiviert ist, wird jedem neuen Benutzer diese erforderliche Aktion zugewiesen.
+disabledOff=Deaktiviert aus
+passwordPolicy=Passwortrichtlinien
+noPasswordPolicies=Keine Passwortrichtlinien definiert
+noPasswordPoliciesInstructions=Sie haben keine Passwortrichtlinien zu diesem Bereich hinzugefügt. Fügen Sie eine Richtlinie hinzu, um loszulegen.
+addPolicy=Richtlinie hinzufügen
+otpPolicy=OTP Richtlinie
+webauthnPolicy=Webauthn Richtlinie
+webauthnPasswordlessPolicy=Webauthn Passwordless Richtlinie
+cibaPolicy=CIBA Richtlinie
+addExecutionTitle=Ausführungsschritt hinzufügen
+addExecution=Ausführung hinzufügen
+addSubFlowTitle=Unterablauf hinzufügen
+addSubFlow=Unterablauf hinzufügen
+emptyExecution=Keine Ausführungsschritte vorhanden
+emptyExecutionInstructions=Sie können mit der Definition dieses Ablaufs beginnen, indem Sie einen Teilablauf oder einen Ausführungsschritt hinzufügen.
+bindFlow=Ablauf verknüpfen
+editInfo=Ablauf editieren
+editFlow=Ablauf editieren
+addStep=Ausführungsschritt hinzufügen
+flowDetails=Ablaufdetails
+steps=Ablaufschritte
+requirement=Erforderlich
```

Comment:
English translation says "Requirement" not "Required", so not sure this is correct.

Reply:
I change it to "Anforderung"